utils/utils.tsx

```diff
@@ -1716,6 +1717,16 @@ export function getNextBillingDate() {
     return nextBillingDate.format('MMM D, YYYY');
```

Comment:
With this change, I think `getNextBillingDate` is not used any where else. Maybe we can remove it.